README.md

```diff
@@ -34,13 +34,46 @@ You can automatically generate a clover report file with every PHPUnit run by ad
     </coverage>
 ```
 
-The default configuration does **NOT** push the badge back into the repository. 
-Setting the input **push_badge** in your workflow to true will enable the commit and push process and the badge will be pushed into the repository after it's been generated. In addition, pushing the badge requires you to update your project settings under `Code and automation` > `Actions` > `General` > `Workflow permissions` to `Read and write permissions`.
+# Default configuration
```

Comment:
Maybe "recommended" instead of "default"? Default imo sounds too much like you don't need to configure anything for it to work like this.